plugins/python-build/share/python-build/test_pypy378_download

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
```

Comment:
A script like this is not useful to keep in the codebase since it does a one-off task.

You can see https://github.com/pyenv/pyenv/pull/1692 for scripts that I used to check URLs in all the scripts if you think you can write one that could have lasting value.

Scripts can be added to `python-build\scripts`.